app/main.py

```diff
@@ -1,2 +1,41 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 8
+        self.dict_ = [None for i in range(self.size)]
+
+    def resize(self) -> None:
+        temp_ = [el for el in self.dict_ if el is not None]
+        if len(temp_) > (self.size * (2 / 3)):
```

Comment:
Move this check up and use `len(self)` in it. Also, I would argue that it would be even better to move this check into `__setitem__()` and only call `resize()` when you need, but that's just my preference, I guess.